spec/fixtures/os_release_mint

```diff
@@ -0,0 +1,2 @@
+DISTR_NAME=“Linuxmint”
```

Comment:
not sure if it matters but these are weird quotes `“` and `”`, should probably be `"`